app/code/core/Mage/Catalog/Model/Product/Image.php

```diff
@@ -76,7 +76,8 @@ class Mage_Catalog_Model_Product_Image extends Mage_Core_Model_Abstract
      *
      * @var array
      */
-    protected $_backgroundColor  = array(255, 255, 255);
+    protected $_backgroundColor    = array(255, 255, 255);
+    protected $_backgroundColorStr = 'ffffff';
```

Comment:
is saving a single call to _rgbToString worth creating another property? From my POV its not.

Reply:
I would say its ok, but its a BC break if someone extended this class and relies on the `$_backgroundColor` to be changed.